wcc-contentful/lib/wcc/contentful/model_builder.rb

```diff
@@ -63,6 +63,12 @@ def build_model(typedef)
               raise ArgumentError, 'Wrong Content Type - ' \
                                    "'#{raw.dig('sys', 'id')}' is a #{ct}, expected #{typedef.content_type}"
             end
+            if raw.dig('sys', 'locale').blank?
+              raise ArgumentError, 'Model layer cannot represent "locale=*" entries. ' \
```

Comment:
Here's an important change.  It never made sense to pass a `locale=*` entry to `Page.new`.  So now the model layer expects a localized entry and will throw if it doesn't get one.